Bot/bot.py

```diff
@@ -95,8 +96,18 @@ async def pause(ctx):
 async def ping(ctx): #command name is function name 
     await ctx.send(f'Latency: {round(bot.latency * 1000)}ms')
 
+@bot.command()
+async def startTask(ctx):
+    for i in range(1,6):
+        sleep(25)
+        await ctx.send('Break Time')

```

Comment:
Looks good bro. IF you want to send an auditory signal, you can use
"await ctx.invoke(bot.get_command('alarm'))"